contrib/aws/aws-sdk-go/aws/aws.go

```diff
@@ -124,14 +127,21 @@ func (h *handlers) awsAgent(req *request.Request) string {
 	return "aws-sdk-go"
 }
 
-func (h *handlers) awsOperation(req *request.Request) string {
-	return req.Operation.Name
-}
-
 func (h *handlers) awsRegion(req *request.Request) string {
 	return req.ClientInfo.SigningRegion
 }
 
-func (h *handlers) awsService(req *request.Request) string {
+func getSpanName(req *request.Request) string {
+	awsService := awsServiceFromRequest(req)
+	awsOperation := awsOperationFromRequest(req)
+	getSpanNameV0 := func(awsService string) string { return awsService + ".command" }
+	return awsnamingschema.NewAWSOutboundOp(awsService, awsOperation, getSpanNameV0).GetName()
+}
+
+func awsServiceFromRequest(req *request.Request) string {
 	return req.ClientInfo.ServiceName
 }
+
+func awsOperationFromRequest(req *request.Request) string {
```

Comment:
+1 on removing from the handler, but I think the `FromRequest` is redundant. The old names are fine.